kubectl/kubectl.bash

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# If there is no current context, get one.
+if [[ $(kubectl config current-context 2> /dev/null) == "" ]]; then
+	cluster=$(gcloud config get-value container/cluster 2> /dev/null)
+	zone=$(gcloud config get-value compute/zone 2> /dev/null)
+	function var_usage() {
+		cat <<EOF
+No cluster is set. To set the cluster (and the zone where it is found), set the environment variables
```

Comment:
kubectl has no concept of GKE and cannot help you configure it to point to a GKE cluster.